mkdocs.yml

```diff
@@ -132,6 +136,9 @@ plugins:
       custom_templates: templates
   - mkdocs-opensafely-databuilder
   - macros
+  - multirepo:
+      cleanup: true
```

Comment:
suggestion: Maybe set this `cleanup` value to be specified as an [environment variable](https://www.mkdocs.org/user-guide/configuration/#environment-variables) with a default of `false`? And then setting it to `true` only for the GitHub Actions build.

> Note: Cleanup should be set to `false` when developing (i.e., when calling `mkdocs serve`). This will prevent importing repos multiple times with livereload.

https://github.com/jdoiro3/mkdocs-multirepo-plugin/blob/4f7820717fcf687576075ec14813ecc71de06535/README.md#setup

Reply:
Done :)